docs/Build.md

```diff
@@ -17,64 +17,9 @@ To maintain a small built bundle size, we take advantage of [Webpack's treeshaki
 
 Webpack (and it's plugins) are highly customisable. This section will detail the choices we have made around how and what is built, where it is built to, and the roles and responsibilities of the various plugins which enable this.
 
-#### Webpack aliases
```

Comment:
The changes to this file seem unrelated to the i8ln (and this PR is already massive).

Reply:
While I agree - they have come in because the changes needed to support `jest` with module mapping and typescript. So the "path/alias" resolution is now completed via `tsconfig` and such this section became out of date.